app/src/androidTest/kotlin/at/bitfire/davdroid/resource/LocalCalendarStoreTest.kt

```diff
@@ -94,24 +95,17 @@ class LocalCalendarStoreTest {
                 Calendars._SYNC_ID to 999,
                 Calendars.CALENDAR_DISPLAY_NAME to "displayName",
             )
+        )!!.asSyncAdapter(account)
 
-            uri = providerClient.insert(
-                Calendars.CONTENT_URI.asSyncAdapter(account),
-                values
-            )!!.asSyncAdapter(account)
-        }
-        return uri!!
-    }
-
-    private fun verifyOwnerAccountIs(expectedOwnerAccount: String) = provider.use {
-        it.query(
+    private fun verifyOwnerAccountIs(provider: ContentProviderClient, expectedOwnerAccount: String) {
+        provider.query(
             calendarUri,
             arrayOf(Calendars.OWNER_ACCOUNT),
             "${Calendars.ACCOUNT_NAME}=?",
             arrayOf(account.name),
             null
         )!!.use { cursor ->
-            cursor.moveToNext()
+            Assume.assumeTrue(cursor.moveToFirst())
```

Comment:
I wonder whether we really want to use Assume if we still haven't tried all other methods (like making the flow hot).

The danger is that we don't know what's going on anymore, and at some point the test may even start to be always skipped (because the assumption is always false). Then it would be quite useless.

I'm for adding the Assume in a another PR when the content provider change doesn't fix it.